### ISSUE TITLE
Clarification

### ISSUE THREAD

**Issue by claasch** [stefanbund/SDLC-GITHUB#2]
Several groups have discussed this project and it appears there is some confusion for the first paper.  Just to make sure, the objective of the first paper is to focus on the entertainment industry as it exists today. Each group will have a slanted focus in their area of specialization. We are not yet covering our new application and how it could bypass traditional media for content distribution and analytics.

**Comment by stefanbund** [comment]
Correct, only do the problem domain analysis. Do not suggest any new applications. When it comes to your industry specialization, you can do this minimally, giving emphasis to a full discussion of the problem domain. 

From Stefan
